integration/cmd_list_test.go

```diff
@@ -35,6 +35,13 @@ func TestGenerateAnAppWithStargateWithListAndVerify(t *testing.T) {
 		)),
```

Comment:
For intergration tests, we can also add a case for the `packet` command that can have fields

Reply:
added on [`1e60bc3`](https://github.com/tendermint/starport/pull/1495/commits/1e60bc3dce429b164a3fcd2ec17a6ba36a4f8833)